code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -68,6 +68,12 @@
 	results = list(/datum/reagent/medicine/synthflesh = 3)
 	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/carbon = 1, /datum/reagent/medicine/styptic_powder = 1)
 
+/datum/chemical_reaction/synthflesh2
```

Comment:
This can be a subtype of `/datum/chemical_reaction/synthflesh` and have the name and results rows removed.

Reply:
Thank you Ghommie I did not know I could do that, done.